frontend/test/__support__/sample_database_fixture.ts

```diff
@@ -177,5 +178,7 @@ export function makeMetadata(
 
   metadata.fields = normalizeFields(metadata.fields);
 
-  return getMetadata({ entities: metadata });
+  return getMetadata(
+    createMockState({ entities: metadata as unknown as EntitiesState }),
+  );
```

Comment:
This is far from perfect, but it doesn't feel valuable to fix it "in the right way" as we're about to remove this whole file for good